build.gradle

```
@@ -46,7 +46,7 @@ dependencies {
 	compile("com.googlecode.json-simple:json-simple:1.1.1") {
 	   exclude group: 'junit'
 	}
-	compile group: 'com.vmware.singleton', name: 'singleton-i18n-patterns-core', version: '+'	
+	compile group: 'com.vmware.singleton', name: 'singleton-i18n-patterns-core', version: '0.5.1'
```

Please check this <a href='https://docs.gradle.org/current/userguide/single_versions.html'>Declaring Versions and Ranges</a>
A Maven-style version range: e.g. [1.0,), [1.1, 2.0), (1.2, 1.5]

The [ and ] symbols indicate an inclusive bound; ( and ) indicate an exclusive bound.

When the upper or lower bound is missing, the range has no upper or lower bound.

The symbol ] can be used instead of ( for an exclusive lower bound, and [ instead of ) for exclusive upper bound. e.g ]1.0, 2.0[
```suggestion
	compile group: 'com.vmware.singleton', name: 'singleton-i18n-patterns-core', version: '[0,)'
```